src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/LocationService.kt

```diff
@@ -17,8 +18,24 @@ class LocationService(
           .map { it.copy(description = it.description.formatLocation()) }
           .toList()
 
+  fun getCellsWithCapacityForGroup(agencyId: String, groupName: String, attribute: String?): List<CellWithAttributes> =
+          elite2ApiService.getCellsWithCapacity(agencyId, attribute)
+                  .filter{ (locationGroupService.locationGroupFilter(agencyId, groupName)::test)(it.mapToLocation(agencyId)) }
+                  .toMutableList()
```

Comment:
Is this undone by the toList()?

Reply:
Actually, do we need to make this mutable at all?

Reply:
This is why I love code reviews. No, we don't need to be making into two different kinds of lists :D Have removed.